Utilities/Thread.cpp

```diff
@@ -765,7 +849,11 @@ static const decltype(REG_RAX) reg_table[] =
 };
 
 #define X64REG(context, reg) (&(context)->uc_mcontext.gregs[reg_table[reg]])
+#ifdef __sun
```

Comment:
I feel like the bitrot here will get pretty bad. BSD support I can get behind, but Solaris? The amount of BSD users using RPCS3 would be tiny anyway, but I think Solaris support is a lost cause.